test/local/account_routes.js

```diff
@@ -1178,20 +1182,54 @@ test('/account/login', function (t) {
         t.equal(response.verified, false, 'response indicates account is unverified')
         t.equal(response.verificationMethod, 'email', 'verificationMethod is email')
         t.equal(response.verificationReason, 'signup', 'verificationReason is signup')
+        t.notOk(response.emailSent, 'email sent, not set')
+      })
+    })
+
+    t.test('with `sendEmailIfUnverified` param, true', function (t) {
+      mockRequest.query.sendEmailIfUnverified = true
+      return runTest(route, mockRequest, function (response) {
+        t.equal(mockMailer.sendVerifyCode.callCount, 1, 'mailer.sendVerifyCode was called')
+        t.equal(mockMailer.sendVerifyLoginEmail.callCount, 0, 'mailer.sendVerifyLoginEmail was not called')
+        t.equal(mockMailer.sendNewDeviceLoginNotification.callCount, 0, 'mailer.sendNewDeviceLoginNotification was not called')
+        t.equal(response.verified, false, 'response indicates account is unverified')
+        t.equal(response.verificationMethod, 'email', 'verificationMethod is email')
+        t.equal(response.verificationReason, 'signup', 'verificationReason is signup')
+        t.equal(response.emailSent, true, 'email sent')
+      }).then(function () {
+        mockRequest.query.sendEmailIfUnverified = undefined
+        mockMailer.sendVerifyCode.reset()
+      })
+    })
+
+    t.test('with `sendEmailIfUnverified` param, false', function (t) {
```

Comment:
Ack, looks like github diff decided not to show it. There was a test case for this, https://github.com/mozilla/fxa-auth-server/pull/1440/files#diff-fcdd8be5dbf433fe907739c4a898fceeR1177